custom/GLideN64/mupenplus/CommonAPIImpl_mupenplus.cpp

```diff
@@ -94,9 +98,10 @@ void PluginAPI::FindPluginPath(wchar_t * _strPath)
 		_getWSPath(path, _strPath);
 	}
 #elif defined(OS_MAC_OS_X)
+	#define MAXPATHLEN 256
```

Comment:
I dont like this one. I could offer moving it to the mupen next shared header and we call it MAX_PATH like the other platforms?

Reply:
This one is a direct copy of what you find here https://github.com/libretro/mupen64plus-libretro-nx/blob/develop/GLideN64/src/mupenplus/CommonAPIImpl_mupenplus.cpp#L110

Reply:
OK I pushed a commit to use PATH_MAX_LENGTH